### DISCUSSION TITLE
New documentation site! https://oxide.md

### DISCUSSION THREAD

**Discussion by Feel-ix-343** [Feel-ix-343/markdown-oxide#151] [Announcements]
I have just finished an initial version of markdown-oxide's new docs site, https://oxide.md

Hopefully, this makes it easier for prospective users to decide of markdown-oxide is for them and set up the PKM.

**Comment by luigir-it** [comment]
Hi. Is the site supposed to be online yet? I can't access it.

**Reply by Feel-ix-343** [reply]
That's odd, it seems to be getting visitors.

https://oxide.md

**Reply by luigir-it** [reply]
My bad, I forgot my dns blocks newly registered domains :man_facepalming:
